From b7afcd51442864bf672e46ddee354da6afd8242c Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Thu, 24 Aug 2017 15:52:04 +0100
Subject: [PATCH 5/6] condor_utils: add include for YourString in tokener.h

This means that a specific header include order is not required.
---
 src/condor_utils/tokener.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/condor_utils/tokener.h b/src/condor_utils/tokener.h
index 614418bb53..7ab97db566 100644
--- a/src/condor_utils/tokener.h
+++ b/src/condor_utils/tokener.h
@@ -19,6 +19,7 @@
 
 #ifndef __TOKENER__
 #define __TOKENER__
+#include "MyString.h"
 
 
 // this is a simple tokenizer class for parsing keywords out of a line of text
-- 
2.13.5

